client/packages/host/src/api/hooks/sync/updateSyncSettings.ts

```diff
@@ -0,0 +1,12 @@
+import { useMutation, useQueryClient } from '@openmsupply-client/common';
+import { useHostApi } from '../utils/useHostApi';
+
+export const useUpdateSyncSettings = () => {
+  const queryClient = useQueryClient();
+  const api = useHostApi();
+  return useMutation(api.update, {
+    onSuccess: () => {
+      queryClient.invalidateQueries(api.keys.updateSyncSettings());
```

Comment:
the cache key that you need to update is `api.keys.syncSettings()` rather than this one. Not that it matters.. since the `cacheTime` is `0` anyway. basically, you don't need the `onSuccess`

Reply:
🙏 forgot about this again: 70a49f5ee7ffbd237340e0ed8fbac838c5d9afc8